src/index.js

```diff
@@ -19,6 +20,7 @@ export const Error = ErrorSource
 export const Heading = HeadingSource
 export const Icon = IconSource
 export const Input = InputSource
+export const List = ListSource
 export const Markdown = MarkdownSource
 export const Paragraph = ParagraphSource
 export const RadioGroup = RadioGroupSource
```

Comment:
Add `List` component to API so that consuming projects can do `yarn upgrade egghead-ui` and then `import {List} from 'egghead-ui'`